packages/package-generator/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "package-generator",
+  "version": "1.3.0",
+  "main": "./lib/main",
+  "description": "Generates and opens a new sample package, language, or syntax theme.",
+  "license": "MIT",
+  "activationCommands": {
+    "atom-workspace": [
+      "package-generator:generate-package",
+      "package-generator:generate-language-package",
+      "package-generator:generate-syntax-theme"
+    ]
+  },
+  "dependencies": {
+    "fs-plus": "^3.0.0",
+    "temp": "^0.8.1",
+    "underscore-plus": "^1.0.0"
+  },
+  "repository": "https://github.com/pulsar-edit/package-generator",
```

Could you update this?
```suggestion
  "repository": "https://github.com/pulsar-edit/pulsar",
```